### PR TITLE
feat: LT-1109 - Case activity search

### PR DESCRIPTION
**Pull request by a-bek**

### CODE REVIEW

File: cases/views/main.py [hunk 1]
```diff
@@ -113,15 +114,32 @@ def get(self, request, **kwargs):
...
         can_set_done = can_set_done and (is_system_queue and user_assigned_queues) or not is_system_queue
 
+        activity_filters = get_activity_filters(request, case_id)
+
+        def make_options(values):
+            return [Option(option["key"], option["value"]) for option in values]
+
+        filters = FiltersBar(
+            [
+                Select(name="user_id", title="User", options=make_options(activity_filters["users"])),
```
Comment (andytaylorkainos) [inline cases/views/main.py:124]: will all of these titles will need to be added to content?

Comment (a-bek) [reply cases/views/main.py:124]: Will add them now :)

File: ui_automation_tests/pages/application_page.py [hunk 2]
```diff
@@ -274,3 +277,19 @@ def click_on_case_link(self, case_id):
 
     def get_case_queues(self):
         return self.driver.find_element_by_id(self.CASE_QUEUES_ID)
+
+    def click_show_filters_link(self):
```
Comment (jfaracik) [inline ui_automation_tests/pages/application_page.py:281]: Theres already a function for this in functions.py

Comment (a-bek) [reply ui_automation_tests/pages/application_page.py:281]: Cool ive used it within this method.

Comment (jfaracik) [reply ui_automation_tests/pages/application_page.py:281]: There isn't much point in writing a wrapper for the method, better just to call it directly! :)

**Pull request merged**